### PR TITLE
design and implementation of booking

### PR DESCRIPTION
**Pull request by afyacy**
- Designed basic booking component
-  Accepts guest name, email, event name, company email, start and end date with time
- API stores data in the database

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -28,8 +28,8 @@ model User {
 
 model Event {
   eventId     Int       @default(autoincrement()) @id
-  startDate   DateTime
-  endDate     DateTime
+  startDate   String
+  endDate     String
```
Comment (simbamentor) [inline prisma/schema.prisma:32]: why did you change it to a string? `DateTime` looks like the correct format for me

File: pages/api/booking.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+export default async function handler (request, response) {
+  await prisma.attendee.create({
```
[the remark this answers is not in the document]
Comment (afyacy) [reply pages/api/booking.js:6]: done :)
